src/ui/macstr_ui.py

```diff
@@ -0,0 +1,201 @@
+        self.radioButton_4.setGeometry(QtCore.QRect(280, 210, 106, 23))
+        self.radioButton_4.setObjectName("radioButton_4")
+
+        self.radioButton_5 = QtWidgets.QRadioButton(self.centralwidget)
+        self.radioButton_5.setGeometry(QtCore.QRect(640, 210, 106, 23))
+        self.radioButton_5.setObjectName("radioButton_5")
+
+        self.radioButton_6 = QtWidgets.QRadioButton(self.centralwidget)
+        self.radioButton_6.setGeometry(QtCore.QRect(530, 210, 106, 23))
+        self.radioButton_6.setObjectName("radioButton_6")
+
+        self.radioButton_7 = QtWidgets.QRadioButton(self.centralwidget)
+        self.radioButton_7.setGeometry(QtCore.QRect(140, 460, 106, 23))
+        self.radioButton_7.setObjectName("radioButton_7")
+
+        self.radioButton_8 = QtWidgets.QRadioButton(self.centralwidget)
+        self.radioButton_8.setGeometry(QtCore.QRect(30, 460, 106, 23))
+        self.radioButton_8.setObjectName("radioButton_8")
+
+        self.radioButton_9 = QtWidgets.QRadioButton(self.centralwidget)
+        self.radioButton_9.setGeometry(QtCore.QRect(390, 460, 106, 23))
+        self.radioButton_9.setObjectName("radioButton_9")
+
+        self.radioButton_10 = QtWidgets.QRadioButton(self.centralwidget)
+        self.radioButton_10.setGeometry(QtCore.QRect(280, 460, 106, 23))
+        self.radioButton_10.setObjectName("radioButton_10")
+
+        self.radioButton_11 = QtWidgets.QRadioButton(self.centralwidget)
+        self.radioButton_11.setGeometry(QtCore.QRect(640, 460, 106, 23))
+        self.radioButton_11.setObjectName("radioButton_11")
+
+        self.radioButton_12 = QtWidgets.QRadioButton(self.centralwidget)
+        self.radioButton_12.setGeometry(QtCore.QRect(530, 460, 106, 23))
+        self.radioButton_12.setObjectName("radioButton_12")
+
+        # ------------------------------
+        # Refresh button
+        # -- used to reload the images
+        # ------------------------------
+        self.pushButton = QtWidgets.QPushButton(self.centralwidget)
+        self.pushButton.setGeometry(QtCore.QRect(330, 500, 83, 25))
+        self.pushButton.setObjectName("pushButton")
+        MainWindow.setCentralWidget(self.centralwidget)
+
+        self.menubar = QtWidgets.QMenuBar(MainWindow)
+        self.menubar.setGeometry(QtCore.QRect(0, 0, 800, 22))
+        self.menubar.setObjectName("menubar")
+        MainWindow.setMenuBar(self.menubar)
+
+        self.statusbar = QtWidgets.QStatusBar(MainWindow)
+        self.statusbar.setObjectName("statusbar")
+        MainWindow.setStatusBar(self.statusbar)
+
+        self.toolBar = QtWidgets.QToolBar(MainWindow)
+        self.toolBar.setObjectName("toolBar")
+        MainWindow.addToolBar(QtCore.Qt.TopToolBarArea, self.toolBar)
+
+        self.retranslateUi(MainWindow)
+        QtCore.QMetaObject.connectSlotsByName(MainWindow)
+
+    def retranslateUi(self, MainWindow):
+        """Renaming of the GUI buttons"""
+        _translate = QtCore.QCoreApplication.translate
```

Comment:
Love the inclusion of translation possibilities! Are any additional files necessary to make sure this works as expected?

Reply:
No that I know off. All of this code has been generated by using the `PyQT Designer` I can check